README.rst

```diff
@@ -455,6 +453,13 @@ Meta-templates for generating Cookiecutter project templates.
 
 .. _`cookiecutter-template`: https://github.com/eviweb/cookiecutter-template
 
+Ansible
+~~
+* `cookiecutter-ansible-role`_: A template to create ansible roles. Forget about file creation and focus on actions.
```

Comment:
Please extend the `~` to the end of heading and add a new line to fix the formatting?